src/components-styled/kpi-tile.tsx

```diff
@@ -1,13 +1,15 @@
 import { Box, Spacer } from './base';
-import { Text, Heading } from './typography';
+import { Heading } from './typography';
 import { Tile } from './layout';
 import { MetadataProps, Metadata } from './metadata';
+import { DataWarning } from '~/components/dataWarning';
 
 interface KpiTileProps {
   title: string;
   description?: string;
   children: React.ReactNode;
   metadata: MetadataProps;
+  dataWarning?: boolean /* TODO: remove this temporary attribute when it is not used anymore */;
```

Comment:
This doesn't read like a boolean.
```suggestion
  showDataWarning?: boolean /* TODO: remove this temporary attribute when it is not used anymore */;
```